packages/okta-adapter/src/definitions/deploy/deploy.ts

```diff
@@ -804,6 +804,51 @@ const createCustomizations = (): Record<string, InstanceDeployApiDefinitions> =>
         },
       },
     },
+    OAuth2Scope: {
+      requestsByAction: {
+        customizations: {
+          add: [
+            {
+              request: {
+                endpoint: {
+                  path: '/api/v1/authorizationServers/{authServerId}/scopes',
```

Comment:
nit (same for all 3 requests):
```suggestion
                  path: '/api/v1/authorizationServers/{parent_id}/scopes',
```
And this way there's no need to specify the context.
See [ref](https://github.com/salto-io/salto/blob/b987c68615c0c50fa8e667b81da0c40c2f6939a2/packages/adapter-components/src/deployment/definition_helpers.ts#L12)